### PR TITLE
Fixed #4 and maven dep

### LINKED ISSUE #4: ServerTap does not handle /reload properly
If you do `/reload confirm` you get this mess:

```
> reload confirm
[20:19:13 INFO]: CONSOLE: Please note that this command is not supported and may cause issues when using some plugins.
[20:19:13 INFO]: CONSOLE: If you encounter any issues please use the /stop command to restart your server.
[20:19:13 INFO]: Debug logging is disabled
[20:19:13 INFO]: Server Ping Player Sample Count: 12
[20:19:13 INFO]: Using 4 threads for Netty based IO
[20:19:13 INFO]: -------- World Settings For [world] --------
[20:19:13 INFO]: View Distance: 10
[20:19:13 INFO]: Allow Zombie Pigmen to spawn from portal blocks: true
[20:19:13 INFO]: Arrow Despawn Rate: 1200 Trident Respawn Rate:1200
[20:19:13 INFO]: Item Merge Radius: 2.5
[20:19:13 INFO]: Item Despawn Rate: 6000
[20:19:13 INFO]: Zombie Aggressive Towards Villager: true
[20:19:13 INFO]: Nerfing mobs spawned from spawners: false
[20:19:13 INFO]: Hopper Transfer: 8 Hopper Check: 1 Hopper Amount: 1
[20:19:13 INFO]: Experience Merge Radius: 3.0
[20:19:13 INFO]: Mob Spawn Range: 8
[20:19:13 INFO]: Cactus Growth Modifier: 100%
[20:19:13 INFO]: Cane Growth Modifier: 100%
[20:19:13 INFO]: Melon Growth Modifier: 100%
[20:19:13 INFO]: Mushroom Growth Modifier: 100%
[20:19:13 INFO]: Pumpkin Growth Modifier: 100%
[20:19:13 INFO]: Sapling Growth Modifier: 100%
[20:19:13 INFO]: Beetroot Growth Modifier: 100%
[20:19:13 INFO]: Carrot Growth Modifier: 100%
[20:19:13 INFO]: Potato Growth Modifier: 100%
[20:19:13 INFO]: Wheat Growth Modifier: 100%
[20:19:13 INFO]: NetherWart Growth Modifier: 100%
[20:19:13 INFO]: Vine Growth Modifier: 100%
[20:19:13 INFO]: Cocoa Growth Modifier: 100%
[20:19:13 INFO]: Bamboo Growth Modifier: 100%
[20:19:13 INFO]: SweetBerry Growth Modifier: 100%
[20:19:13 INFO]: Kelp Growth Modifier: 100%
[20:19:13 INFO]: Entity Activation Range: An 32 / Mo 32 / Ra 48 / Mi 16 / Tiv true
[20:19:13 INFO]: Entity Tracking Range: Pl 48 / An 48 / Mo 48 / Mi 32 / Other 64
[20:19:13 INFO]: Custom Map Seeds:  Village: 10387312 Desert: 14357617 Igloo: 14357618 Jungle: 14357619 Swamp: 14357620 Monument: 10387313 Ocean: 14357621 Shipwreck: 165745295 Slime: 987234911
[20:19:13 INFO]: Max TNT Explosions: 100
[20:19:13 INFO]: Tile Max Tick Time: 50ms Entity max Tick Time: 50ms
[20:19:13 INFO]: -------- World Settings For [world_nether] --------
[20:19:13 INFO]: View Distance: 10
[20:19:13 INFO]: Allow Zombie Pigmen to spawn from portal blocks: true
[20:19:13 INFO]: Arrow Despawn Rate: 1200 Trident Respawn Rate:1200
[20:19:13 INFO]: Item Merge Radius: 2.5
[20:19:13 INFO]: Item Despawn Rate: 6000
[20:19:13 INFO]: Zombie Aggressive Towards Villager: true
[20:19:13 INFO]: Nerfing mobs spawned from spawners: false
[20:19:13 INFO]: Hopper Transfer: 8 Hopper Check: 1 Hopper Amount: 1
[20:19:13 INFO]: Experience Merge Radius: 3.0
[20:19:13 INFO]: Mob Spawn Range: 8
[20:19:13 INFO]: Cactus Growth Modifier: 100%
[20:19:13 INFO]: Cane Growth Modifier: 100%
[20:19:13 INFO]: Melon Growth Modifier: 100%
[20:19:13 INFO]: Mushroom Growth Modifier: 100%
[20:19:13 INFO]: Pumpkin Growth Modifier: 100%
[20:19:13 INFO]: Sapling Growth Modifier: 100%
[20:19:13 INFO]: Beetroot Growth Modifier: 100%
[20:19:13 INFO]: Carrot Growth Modifier: 100%
[20:19:13 INFO]: Potato Growth Modifier: 100%
[20:19:13 INFO]: Wheat Growth Modifier: 100%
[20:19:13 INFO]: NetherWart Growth Modifier: 100%
[20:19:13 INFO]: Vine Growth Modifier: 100%
[20:19:13 INFO]: Cocoa Growth Modifier: 100%
[20:19:13 INFO]: Bamboo Growth Modifier: 100%
[20:19:13 INFO]: SweetBerry Growth Modifier: 100%
[20:19:13 INFO]: Kelp Growth Modifier: 100%
[20:19:13 INFO]: Entity Activation Range: An 32 / Mo 32 / Ra 48 / Mi 16 / Tiv true
[20:19:13 INFO]: Entity Tracking Range: Pl 48 / An 48 / Mo 48 / Mi 32 / Other 64
[20:19:13 INFO]: Custom Map Seeds:  Village: 10387312 Desert: 14357617 Igloo: 14357618 Jungle: 14357619 Swamp: 14357620 Monument: 10387313 Ocean: 14357621 Shipwreck: 165745295 Slime: 987234911
[20:19:13 INFO]: Max TNT Explosions: 100
[20:19:13 INFO]: Tile Max Tick Time: 50ms Entity max Tick Time: 50ms
[20:19:13 INFO]: -------- World Settings For [world_the_end] --------
[20:19:13 INFO]: View Distance: 10
[20:19:13 INFO]: Allow Zombie Pigmen to spawn from portal blocks: true
[20:19:13 INFO]: Arrow Despawn Rate: 1200 Trident Respawn Rate:1200
[20:19:13 INFO]: Item Merge Radius: 2.5
[20:19:13 INFO]: Item Despawn Rate: 6000
[20:19:13 INFO]: Zombie Aggressive Towards Villager: true
[20:19:13 INFO]: Nerfing mobs spawned from spawners: false
[20:19:13 INFO]: Hopper Transfer: 8 Hopper Check: 1 Hopper Amount: 1
[20:19:13 INFO]: Experience Merge Radius: 3.0
[20:19:13 INFO]: Mob Spawn Range: 8
[20:19:13 INFO]: Cactus Growth Modifier: 100%
[20:19:13 INFO]: Cane Growth Modifier: 100%
[20:19:13 INFO]: Melon Growth Modifier: 100%
[20:19:13 INFO]: Mushroom Growth Modifier: 100%
[20:19:13 INFO]: Pumpkin Growth Modifier: 100%
[20:19:13 INFO]: Sapling Growth Modifier: 100%
[20:19:13 INFO]: Beetroot Growth Modifier: 100%
[20:19:13 INFO]: Carrot Growth Modifier: 100%
[20:19:13 INFO]: Potato Growth Modifier: 100%
[20:19:13 INFO]: Wheat Growth Modifier: 100%
[20:19:13 INFO]: NetherWart Growth Modifier: 100%
[20:19:13 INFO]: Vine Growth Modifier: 100%
[20:19:13 INFO]: Cocoa Growth Modifier: 100%
[20:19:13 INFO]: Bamboo Growth Modifier: 100%
[20:19:13 INFO]: SweetBerry Growth Modifier: 100%
[20:19:13 INFO]: Kelp Growth Modifier: 100%
[20:19:13 INFO]: Entity Activation Range: An 32 / Mo 32 / Ra 48 / Mi 16 / Tiv true
[20:19:13 INFO]: Entity Tracking Range: Pl 48 / An 48 / Mo 48 / Mi 32 / Other 64
[20:19:13 INFO]: Custom Map Seeds:  Village: 10387312 Desert: 14357617 Igloo: 14357618 Jungle: 14357619 Swamp: 14357620 Monument: 10387313 Ocean: 14357621 Shipwreck: 165745295 Slime: 987234911
[20:19:13 INFO]: Max TNT Explosions: 100
[20:19:13 INFO]: Tile Max Tick Time: 50ms Entity max Tick Time: 50ms
[20:19:13 INFO]: [ServerTap] Disabling ServerTap v1.0-SNAPSHOT
[20:19:13 INFO]: Reloading ResourceManager: Default, bukkit
[20:19:13 INFO]: Loaded 6 recipes
[20:19:13 INFO]: Reloading ResourceManager: Default, bukkit
[20:19:13 INFO]: Loaded 6 recipes
[20:19:13 INFO]: [ServerTap] Loading ServerTap v1.0-SNAPSHOT
[20:19:14 INFO]: Server permissions file permissions.yml is empty, ignoring it
[20:19:14 INFO]: [ServerTap] Enabling ServerTap v1.0-SNAPSHOT
[20:19:14 INFO]: [io.javalin.Javalin]
           __                      __ _
          / /____ _ _   __ ____ _ / /(_)____
     __  / // __ `/| | / // __ `// // // __ \
    / /_/ // /_/ / | |/ // /_/ // // // / / /
    \____/ \__,_/  |___/ \__,_//_//_//_/ /_/

        https://javalin.io/documentation

[20:19:14 INFO]: [org.eclipse.jetty.util.log] Logging initialized @119728ms to org.eclipse.jetty.util.log.Slf4jLog
[20:19:14 INFO]: [io.javalin.Javalin] Starting Javalin ...
[20:19:14 ERROR]: [io.javalin.Javalin] Failed to start Javalin
[20:19:14 INFO]: [io.javalin.Javalin] Stopping Javalin ...
[20:19:14 INFO]: [io.javalin.Javalin] Javalin has stopped
[20:19:14 ERROR]: Error occurred while enabling ServerTap v1.0-SNAPSHOT (Is it up to date?)
java.lang.RuntimeException: Port already in use. Make sure no other process is using port 4567 and try again.
	at io.javalin.Javalin.start(Javalin.java:182) ~[?:?]
	at io.javalin.Javalin.start(Javalin.java:150) ~[?:?]
	at io.servertap.PluginEntrypoint.onEnable(PluginEntrypoint.java:55) ~[?:?]
	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:263) ~[patched_1.15.2.jar:git-Paper-89]
	at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:337) ~[patched_1.15.2.jar:git-Paper-89]
	at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:435) ~[patched_1.15.2.jar:git-Paper-89]
	at org.bukkit.craftbukkit.v1_15_R1.CraftServer.enablePlugin(CraftServer.java:470) ~[patched_1.15.2.jar:git-Paper-89]
	at org.bukkit.craftbukkit.v1_15_R1.CraftServer.enablePlugins(CraftServer.java:384) ~[patched_1.15.2.jar:git-Paper-89]
	at org.bukkit.craftbukkit.v1_15_R1.CraftServer.reload(CraftServer.java:862) ~[patched_1.15.2.jar:git-Paper-89]
	at org.bukkit.Bukkit.reload(Bukkit.java:610) ~[patched_1.15.2.jar:git-Paper-89]
	at org.bukkit.command.defaults.ReloadCommand.execute(ReloadCommand.java:54) ~[patched_1.15.2.jar:git-Paper-89]
	at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:159) ~[patched_1.15.2.jar:git-Paper-89]
	at org.bukkit.craftbukkit.v1_15_R1.CraftServer.dispatchCommand(CraftServer.java:742) ~[patched_1.15.2.jar:git-Paper-89]
	at org.bukkit.craftbukkit.v1_15_R1.CraftServer.dispatchServerCommand(CraftServer.java:704) ~[patched_1.15.2.jar:git-Paper-89]
	at net.minecraft.server.v1_15_R1.DedicatedServer.handleCommandQueue(DedicatedServer.java:469) ~[patched_1.15.2.jar:git-Paper-89]
	at net.minecraft.server.v1_15_R1.DedicatedServer.b(DedicatedServer.java:431) ~[patched_1.15.2.jar:git-Paper-89]
	at net.minecraft.server.v1_15_R1.MinecraftServer.a(MinecraftServer.java:1112) ~[patched_1.15.2.jar:git-Paper-89]
	at net.minecraft.server.v1_15_R1.MinecraftServer.run(MinecraftServer.java:934) ~[patched_1.15.2.jar:git-Paper-89]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_181]
Caused by: java.io.IOException: Failed to bind to 0.0.0.0/0.0.0.0:4567
	at org.eclipse.jetty.server.ServerConnector.openAcceptChannel(ServerConnector.java:346) ~[?:?]
	at org.eclipse.jetty.server.ServerConnector.open(ServerConnector.java:307) ~[?:?]
	at org.eclipse.jetty.server.AbstractNetworkConnector.doStart(AbstractNetworkConnector.java:80) ~[?:?]
	at org.eclipse.jetty.server.ServerConnector.doStart(ServerConnector.java:231) ~[?:?]
	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:72) ~[?:?]
	at org.eclipse.jetty.server.Server.doStart(Server.java:385) ~[?:?]
	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:72) ~[?:?]
	at io.javalin.core.JavalinServer.start(JavalinServer.kt:72) ~[?:?]
	at io.javalin.Javalin.start(Javalin.java:172) ~[?:?]
	... 18 more
Caused by: java.net.BindException: Address already in use
	at sun.nio.ch.Net.bind0(Native Method) ~[?:1.8.0_181]
	at sun.nio.ch.Net.bind(Net.java:433) ~[?:1.8.0_181]
	at sun.nio.ch.Net.bind(Net.java:425) ~[?:1.8.0_181]
	at sun.nio.ch.ServerSocketChannelImpl.bind(ServerSocketChannelImpl.java:223) ~[?:1.8.0_181]
	at sun.nio.ch.ServerSocketAdaptor.bind(ServerSocketAdaptor.java:74) ~[?:1.8.0_181]
	at org.eclipse.jetty.server.ServerConnector.openAcceptChannel(ServerConnector.java:342) ~[?:?]
	at org.eclipse.jetty.server.ServerConnector.open(ServerConnector.java:307) ~[?:?]
	at org.eclipse.jetty.server.AbstractNetworkConnector.doStart(AbstractNetworkConnector.java:80) ~[?:?]
	at org.eclipse.jetty.server.ServerConnector.doStart(ServerConnector.java:231) ~[?:?]
	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:72) ~[?:?]
	at org.eclipse.jetty.server.Server.doStart(Server.java:385) ~[?:?]
	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:72) ~[?:?]
	at io.javalin.core.JavalinServer.start(JavalinServer.kt:72) ~[?:?]
	at io.javalin.Javalin.start(Javalin.java:172) ~[?:?]
	... 18 more
[20:19:14 INFO]: [ServerTap] Disabling ServerTap v1.0-SNAPSHOT
[20:19:14 INFO]: CONSOLE: Reload complete.
[20:19:14 INFO]: Timings Reset
>
```

Not 100% sure it's that big of a deal since it's not a supported command, but it might be nice to investigate if anything can be done.

### PR DESCRIPTION
**Pull request by au5ton**

### CODE REVIEW

File: src/main/java/io/servertap/PluginEntrypoint.java [hunk 1]
```diff
@@ -7,6 +7,7 @@
 import io.servertap.api.v1.ServerApi;
 import net.milkbowl.vault.economy.Economy;
 import org.bukkit.Bukkit;
+import org.bukkit.plugin.Plugin;
```
Comment (au5ton) [inline src/main/java/io/servertap/PluginEntrypoint.java:10]: I think this is leftover from IntelliJ and I forgot to remove this

File: pom.xml [hunk 2]
```diff
@@ -27,6 +27,10 @@
             <id>jitpack.io</id>
             <url>https://jitpack.io</url>
         </repository>
+        <repository>
+            <id>central</id>
+            <url>https://repo1.maven.org/maven2/</url>
+        </repository>
```
Comment (phybros) [inline pom.xml:33]: What's this for?

Comment (au5ton) [reply pom.xml:33]: I couldn't get `com.fasterxml.jackson.core.jackson-databind` to satisfy without it.

Comment (phybros) [reply pom.xml:33]: Weird. It just comes down from maven central automatically on my machine. And in Github actions...

Maybe it's a weird versioning thing. Probably fine to explicitly include it here

**Pull request merged**